.travis.yml

```diff
@@ -7,11 +7,11 @@ env:
     - CVC4_URL="http://cvc4.cs.stanford.edu/downloads/builds/x86_64-linux-opt/cvc4-1.5-x86_64-linux-opt"
     - BOOLECTOR_URL="https://github.com/Boolector/boolector/archive/3.1.0.tar.gz"
   matrix:
-    - RACKET_VERSION=6.9
-    - RACKET_VERSION=7.0
-    - RACKET_VERSION=7.2
+    - RACKET_VERSION=6.9  # Our minimum supported version
+    - RACKET_VERSION=RELEASE
```

Comment:
Wouldn't this get subsumed by RELEASE ALL_SOLVERS=1?

Reply:
Yep, but we run both just to get faster feedback on failures (the ALL_SOLVERS build takes a while).

Reply:
Oh interesting. I thought that when a test fails it fails the whole task immediately. That's not the case?

Reply:
Yeah you're right, I'll change it and we can save Travis a bit of money :-)

Reply:
Oh, never mind, Travis doesn't abort when a test step fails -- it will keep running the rest of the test steps ([example](https://travis-ci.org/emina/rosette/jobs/498011107?utm_medium=notification&utm_source=github_status)). So let's leave it as-is.